ZIGSIMPlus/Info.plist

```diff
@@ -19,17 +19,17 @@
 	<key>CFBundleShortVersionString</key>
 	<string>2.0.0</string>
 	<key>CFBundleVersion</key>
-	<string>1.0.0</string>
+	<string>1.0.1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>NFCReaderUsageDescription</key>
-	<string>This app scans NFC in NFC Reader.</string>
+	<string>This app scans NFC in NFC Reader command to read the data and output them over network.</string>
 	<key>NSCameraUsageDescription</key>
-	<string>This app uses camera in NDI, ARKit and Image Detection.</string>
+	<string>This app uses camera for in NDI, ARKit and Image Detection commands.  NDI command outputs captured images over NDI protocol. ARKit command, tracks data of the device, face and markers with the camera, then output them over network.  Image Detection command detects face, rectangle etc in captured images and output the data over network.</string>
```

Comment:
Isn't `for in` weird?
Too many spaces Before `NDI command outputs...` and `Image Detection command detects ...`.
`then output them over network` -> `then output"s" them over network`
`etc` -> `etc.`

Reply:
thanks! fixed in 30364d9

Reply:
👍